lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,236 @@
+    width: 100%;
+  }
+`
+const TextWIcon = styled.div`
+  align-items: center;
+  display: flex;
+  gap: 7px;
+  justify-content: left;
+  width: 250px;
+
+  svg {
+    flex-shrink: 0;
+  }
+`
+
+const TripDetailWithIcon = styled(TextWIcon)`
+  gap: 12px;
+  width: 273px;
+
+  svg {
+    width: 16px;
+  }
+
+  &:last-of-type {
+    align-items: flex-start;
+
+    svg {
+      margin-top: 2px;
+    }
+  }
+`
+
+const TripDetailsList = styled.ul`
+  align-items: start;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 15px;
+  justify-content: center;
+  padding: 0;
+`
+
+const ToggleNotificationButton = styled.button`
+  background: transparent;
+  border: none;
+  font-style: italic;
+  padding: 0;
+  text-decoration: underline;
+`
+
 const TripSummaryPane = ({
-  monitoredTrip
-}: MonitoredTripProps): JSX.Element => {
+  from,
+  handleTogglePauseMonitoring,
+  intl,
+  monitoredTrip,
+  pendingRequest,
+  to
+}: Props): JSX.Element => {
   const { itinerary, leadTimeInMinutes } = monitoredTrip
-
   if (!itinerary) {
     return (
       <div>
         <FormattedMessage id="common.itineraryDescriptions.noItineraryToDisplay" />
       </div>
     )
   } else {
-    const days = (
-      <FormattedList
-        type="conjunction"
-        value={dayFieldsToArray(monitoredTrip).map((d) => (
-          <FormattedDayOfWeek day={d} key={d} />
-        ))}
-      />
-    )
+    const days = dayFieldsToArray(monitoredTrip)
+
+    const testHandle = () => {
+      if (handleTogglePauseMonitoring) {
+        handleTogglePauseMonitoring()
+      }
+    }
+
+    const notificationLabel = intl.formatMessage({
+      id: 'components.ExistingAccountDisplay.notifications'
+    })
+    const durationLabel = intl.formatMessage({
+      id: 'components.TripSummaryPane.timeAndDuration'
+    })
+    const monitoredDaysLabel = intl.formatMessage({
+      id: 'components.TripSummaryPane.monitoredTripDays'
+    })
 
     return (
-      <>
-        {/* TODO: use the modern itinerary summary built for trip comparison. */}
-        <TripSummary monitoredTrip={monitoredTrip} />
-        <p>
+      <SavedTripBody>
+        <InvisibleA11yLabel>
           <FormattedMessage
-            id="components.TripSummaryPane.happensOnDays"
-            values={{ days, strong: Strong }}
+            id="components.SavedTripList.fromTo"
+            values={{ from: from?.name, to: to?.name }}
           />
-        </p>
-        <p>
-          {monitoredTrip.isActive ? (
-            <FormattedMessage
-              id="components.TripSummaryPane.notifications"
-              values={{ leadTimeInMinutes, strong: Strong }}
-            />
-          ) : (
-            <FormattedMessage
-              id="components.TripSummaryPane.notificationsDisabled"
-              values={{ leadTimeInMinutes, strong: Strong }}
-            />
-          )}
-        </p>
-      </>
+        </InvisibleA11yLabel>
+        <LocationDetails aria-hidden>
```

Comment:
I would prefer this block to be aligned top, but I'll go with the majority opinion.

Reply:
Here is the aligned top. I think it makes the card look a little uneven but curious what 2nd reviewer thinks:
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/7b4c615c-9be5-4895-8099-d1c352578e98)

Reply:
Yeah I actually think it's not centered enough as is right now